### PR TITLE
(freecad) Updated to remove files and dev version issues {Take 2}

### PR DESCRIPTION
**Pull request by RedBaron2**
@gep13 @AdmiringWorm @luzpaz @adrianinsaval

## Description

This will fix the package from trying to push as a embedded package as it is **not** an embedded package. It happened because  I forgot to update for removing the `zip` files. This will also allow for direct updating of the dev minor version as a separate variable from the update script. This will only be needed if the update helper logic fails.

## Motivation and Context

This will fix issue #1941

## How Has this Been Tested?

Intranet and appveyor

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [X] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [X] I have updated the package description and it is less than 4000 characters.
- [X] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [X] The added/modified package passed install/uninstall in the chocolatey test environment.
- [X] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](add_link_to_original_repository_location)
- [Open Issues](link_to_the_generic_location_of_open_issues) *Add the different issues underneath, and tick those that are fixed in this PR*
  - [X] Issue 1 #1941 
  - [X] Issue 2 #1948 Appveyor Errored on this PR so Opened this PR
- [ ] *Include the link to the opened PR that removes the package from the original location*
- [ ] The [migration guidelines](https://github.com/chocolatey-community/chocolatey-packages/wiki/Package-migration-process) have been followed

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -11,30 +11,31 @@
 # Packages
```
Comment (AdmiringWorm) [inline .github/CODEOWNERS:11]: Other than adding yourself to this file, please do not make any other changes to the file.

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (RedBaron2) [reply .github/CODEOWNERS:11]: FYI: Second off the errors where there before I even added myself too the list.
The last commit had errors
`This CODEOWNERS file contains errors …
CODEOWNERS errors
Unknown owner on line 27: make sure @moebius87 exists and has write access to the repository
*/k9s*                @moebius87
Unknown owner on line 28: make sure @johnstevenson exists and has write access to the repository
*/composer*           @johnstevenson
Unknown owner on line 29: make sure @jetersen exists and has write access to the repository
*/krew*               @jetersen
Unknown owner on line 30: make sure @jetersen exists and has write access to the repository
*/kubelogin*          @jetersen
Unknown owner on line 31: make sure @VuiMuich exists and has write access to the repository
*/ffmpeg-full*        @VuiMuich
Unknown owner on line 32: make sure @dgalbraith exists and has write access to the repository
*/etcd*               @dgalbraith
Unknown owner on line 33: make sure @dgalbraith exists and has write access to the repository
*/octave.portable*    @dgalbraith
Unknown owner on line 34: make sure @hansnull exists and has write access to the repository
*/selenium-gecko-driver* @hansnull @chocolatey-community/chocolatey-team-maintainers
@@ -34,6 +34,7 @@
*/selenium-gecko-driver* @hansnull @chocolatey-community/chocolatey-team-maintainers
*/es*                 @TheCakeIsNaOH
*/selenium-chrome-driver* @chocolatey-community/chocolatey-team-maintainers
*/intunewinapputil*   @chocolatey-community/chocolatey-team-maintainers`

How can alphabetizing the list of packages in the file be a major change?

Comment (AdmiringWorm) [reply .github/CODEOWNERS:11]: > FYI: Second off the errors where there before I even added myself to the list.
> The last commit had errors

I am aware of that, this is due to GitHub not actually supporting automatic review requests from people that do not have write access to the repository.

> How can alphabetizing the list of packages in the file be a major change?

I never said it was a major change, I only said to not make any other changes than adding yourself to the list.

Comment (RedBaron2) [reply .github/CODEOWNERS:11]: Will revert as time allows.
I will hope this gets fixed before this become mandatory.

I request that the list of packages in this file be alphabetized after this PR.

**Comment by RedBaron2** [comment]
@AdmiringWorm 
I reverted the codeoweners file back and added to the bottom. I rechecked the updates to the regex and everything tests fine.

**Review by AdmiringWorm — APPROVED** [review]
LGTM, thank you for making these changes

**Comment by AdmiringWorm** [comment]
@RedBaron2 your changes have been merged, thanks for your contribution 👍

**Comment by luzpaz** [comment]
Thanks for all the work on this y'all!

**Comment by RedBaron2** [comment]
I seem to have skipped a step in this PR. I'll fix that soon. Oops

**Comment by RedBaron2** [comment]
@AdmiringWorm 
The new PR to make this package update has been posted. If you could please take a look.

**Pull request merged**